src/picongpu/include/simulation_defines/unitless/ionizerConfig.unitless

```diff
@@ -22,3 +22,10 @@
 
 #include "particles/ionization/byField/BSI/AlgorithmBSI.hpp"
 #include "particles/ionization/None/AlgorithmNone.hpp"
+
+namespace picongpu
+{
+    /* magic number for ionization probability calculation to reduce computation time */
+    const float_X PROB_THRESH = 0.1;
```

Comment:
Sorry, I rethought the whole thing and came to the conclusion that it's not needed at all. 
Multi-Photon ionization (MPI) would require a Poissonian-process-like way of calculating the probability. 
```
probMPI = 1 - exp( - ionization_rate * time_step)
```
BUT tunneling ionization is no Poissonian process, I think. The models tunneling models then just lose their applicability once the probability for ionization reaches 1.
```
probADK = ionization_rate * time_step
```